Gruntfile.js

```diff
@@ -52,6 +52,7 @@ const bedrockDefaults = {
   customRoutes: 'modules/tinymce/src/core/test/json/routes.json',
   overallTimeout: 180000,
   singleTimeout: 60000,
+  polyfills: [ 'Promise', 'Symbol' ],
```

Comment:
Is that something we should be adding here or is that something that should be part of the bedrock defaults for anything using bedrock (so we don't have to copy this into every repo)

Reply:
See my reply to Spocke above on this one :stuck_out_tongue: (you were also part of the discussion on this, so I'm somewhat surprised by this question):
```
It's more some project might not be using ESLint (I was thinking the ReasonML code base for example), so we thought it was dangerous to turn the promise polyfill on by default. So after talking to the team we decided to make it explicit opt-in.
```